Kevin

kevinor03 opened the pull request.

Grupo10Henry reviewed (changes requested):
Afueeeeeraaaaaaaaa 😂

kevinor03 committed:
base de home

kevinor03 committed:
Merge pull request #19 from Grupo10Henry/Kevin

base de home

lccarrillo247 committed:
Ajustes menores

Grupo10Henry committed:
Merge pull request #21 from Grupo10Henry/Luis

Ajustes menores

lccarrillo247 committed:
Arreglo BrowserRouter y visibilidad AdminNavbar

Grupo10Henry committed:
Merge pull request #22 from Grupo10Henry/Luis

Arreglo BrowserRouter y visibilidad AdminNavbar

lccarrillo247 committed:
Mover carpetas Guada + AdminNavbar V01

Grupo10Henry committed:
Merge pull request #23 from Grupo10Henry/Luis

Mover carpetas Guada + AdminNavbar V01

lccarrillo247 committed:
AdminLogin v01

Grupo10Henry committed:
Merge pull request #24 from Grupo10Henry/Luis

AdminLogin v01

lccarrillo247 committed:
Cambio ruta home

Grupo10Henry committed:
Merge pull request #25 from Grupo10Henry/Luis

Cambio ruta home

lccarrillo247 committed:
Reestructuración carpetas Admin

Grupo10Henry committed:
Merge pull request #26 from Grupo10Henry/Luis

Reestructuración carpetas Admin

lccarrillo247 committed:
Quitar adminnavbar de app

Grupo10Henry committed:
Merge pull request #27 from Grupo10Henry/Luis

Quitar adminnavbar de app

lccarrillo247 committed:
AdminNavbar Estilos v01

Grupo10Henry committed:
Merge pull request #28 from Grupo10Henry/Luis

AdminNavbar Estilos v01

lccarrillo247 committed:
Sección adminNavbar y adminComponents

Grupo10Henry committed:
Merge pull request #29 from Grupo10Henry/Luis

Sección adminNavbar y adminComponents

lccarrillo247 committed:
Esqueleto form CreateEvent

Grupo10Henry committed:
Merge pull request #30 from Grupo10Henry/Luis

Esqueleto form CreateEvent

lccarrillo247 committed:
CreateEvent v01

Grupo10Henry committed:
Merge pull request #31 from Grupo10Henry/Luis

CreateEvent v01

kevinor03 committed:
cambio en contacto y app

kevinor03 committed:
Merge pull request #32 from Grupo10Henry/Kevin

cambio en contacto y app

kevinor03 committed:
cambio en contact

kevinor03 committed:
Merge pull request #33 from Grupo10Henry/Kevin

cambio en contact

JE-FUNES committed:
sign up

JE-FUNES committed:
Merge branch 'develop' into Juli

JE-FUNES committed:
Merge pull request #34 from Grupo10Henry/Juli

sign up

kevinor03 committed:
ordenamiento de cards

kevinor03 committed:
Merge pull request #35 from Grupo10Henry/Kevin

ordenamiento de cards

francorinque committed:
cambie color del navbar, arregle el contacto y le meti mas estilos, rutas del nav corregidas

francorinque committed:
Merge pull request #36 from Grupo10Henry/franco

cambie color del navbar, arregle el contacto y le meti mas estilos, r…

GuadalupeRedondo committed:
footerAdmin

lccarrillo247 committed:
Merge pull request #37 from Grupo10Henry/guada

footerAdmin

JE-FUNES committed:
signUp y Login 1era instancia terminada

falta Google en ambos

JE-FUNES committed:
Merge branch 'develop' into Juli

JE-FUNES committed:
Merge pull request #38 from Grupo10Henry/Juli

signUp y Login 1era instancia terminada. Merge sin errores a Develop.

JE-FUNES committed:
Update App.jsx

habian quedado 3 importaciones duplicadas

JE-FUNES committed:
Update App.jsx

JE-FUNES committed:
login y signup con acceso de prueba

JE-FUNES committed:
Merge branch 'develop' into Juli

JE-FUNES committed:
Merge pull request #39 from Grupo10Henry/Juli

Juli

Grupo10Henry committed:
Update App.jsx

Grupo10Henry committed:
Update App.jsx

JE-FUNES committed:
cart en proceso

JE-FUNES committed:
Merge branch 'develop' of https://github.com/Grupo10Henry/Client into develop

JE-FUNES committed:
Merge pull request #40 from Grupo10Henry/Juli

Update App.jsx

JE-FUNES committed:
Merge branch 'develop' into Juli

JE-FUNES committed:
Update Cart.jsx

carrito con posibilidad de demo listo

JE-FUNES committed:
Merge pull request #41 from Grupo10Henry/Juli

Juli

GuadalupeRedondo committed:
adminFooter

GuadalupeRedondo committed:
Merge pull request #42 from Grupo10Henry/guada

adminFooter

lccarrillo247 committed:
NavBar admin VF

lccarrillo247 committed:
Merge pull request #43 from Grupo10Henry/Luis

NavBar admin VF

JE-FUNES committed:
App.jsx con nuevas rutas y español

solo app, nada mas

JE-FUNES committed:
Merge pull request #44 from Grupo10Henry/Juli

App.jsx con nuevas rutas y español

lccarrillo247 committed:
Footer al final

lccarrillo247 committed:
Merge pull request #46 from Grupo10Henry/Luis

Footer al final

francorinque committed:
cambie de lugar el searchbar, le di estilos y cambie las rutas del nav.

francorinque committed:
texto de los selects responsive

francorinque committed:
Merge pull request #47 from Grupo10Henry/franco

Franco

francorinque committed:
cambio color body, animacion logo y links

francorinque committed:
Merge pull request #48 from Grupo10Henry/franco

cambio color body, animacion logo y links

JE-FUNES committed:
sign up funcionando 100%

JE-FUNES committed:
Merge pull request #49 from Grupo10Henry/Juli

sign up funcionando 100%

JE-FUNES committed:
varios

JE-FUNES committed:
Merge pull request #50 from Grupo10Henry/Juli

varios

JE-FUNES committed:
varios 2

JE-FUNES committed:
Merge pull request #51 from Grupo10Henry/Juli

varios 2

JE-FUNES committed:
cambios

JE-FUNES committed:
sign up final

JE-FUNES committed:
Merge pull request #52 from Grupo10Henry/Juli

Juli

JE-FUNES committed:
Merge pull request #53 from Grupo10Henry/develop

Develop

JE-FUNES committed:
Login y SignUp

ajuste de términos, css, funcionalidades...

JE-FUNES committed:
Merge pull request #54 from Grupo10Henry/Juli

Juli

JE-FUNES committed:
prueba git re-commit

JE-FUNES committed:
Merge pull request #55 from Grupo10Henry/Juli

prueba git re-commit

JE-FUNES committed:
Merge pull request #56 from Grupo10Henry/develop

Develop

kevinor03 committed:
visual de cards con datos

kevinor03 committed:
Merge pull request #57 from Grupo10Henry/Kevin

visual de cards con datos

kevinor03 committed:
visual de cards basica

kevinor03 committed:
Merge pull request #58 from Grupo10Henry/Kevin

visual de cards basica

JE-FUNES committed:
Login y SignUp 100% funcionales

JE-FUNES committed:
Merge pull request #59 from Grupo10Henry/Juli

Juli

JE-FUNES committed:
Merge pull request #60 from Grupo10Henry/develop

Develop

francorinque committed:
colores 100% fijos, filtros ya pueden pedir al back

francorinque committed:
Merge pull request #61 from Grupo10Henry/franco

colores 100% fijos, filtros ya pueden pedir al back

lccarrillo247 committed:
GoogleMaps v01

lccarrillo247 committed:
Merge pull request #62 from Grupo10Henry/Luis

GoogleMaps v01

JE-FUNES committed:
comenzando detail

JE-FUNES committed:
Merge pull request #63 from Grupo10Henry/Juli

Juli

JE-FUNES committed:
Merge pull request #64 from Grupo10Henry/develop

Develop

lccarrillo247 committed:
Remove GoogleApiKey

lccarrillo247 committed:
Merge pull request #65 from Grupo10Henry/Luis

Remove GoogleApiKey

lccarrillo247 committed:
CreateEvent CSS v01

lccarrillo247 committed:
Merge pull request #66 from Grupo10Henry/Luis

CreateEvent CSS v01

francorinque committed:
filtros no se muestra en ruta admin

francorinque committed:
Merge pull request #67 from Grupo10Henry/franco

filtros no se muestra en ruta admin

lccarrillo247 committed:
Create section v01

lccarrillo247 committed:
Merge pull request #68 from Grupo10Henry/Luis

Create section v01

lccarrillo247 committed:
Funcionalidad botón crear evento

lccarrillo247 committed:
Merge pull request #69 from Grupo10Henry/Luis

Funcionalidad botón crear evento

JE-FUNES committed:
reducers y detail

JE-FUNES committed:
Merge pull request #70 from Grupo10Henry/Juli

Juli

JE-FUNES committed:
esquema rápido de detail

JE-FUNES committed:
Merge pull request #71 from Grupo10Henry/Juli

esquema rápido de detail

lccarrillo247 committed:
Users table v01

lccarrillo247 committed:
Merge pull request #72 from Grupo10Henry/Luis

Users table v01

francorinque committed:
estilos a card,estado de redux para los eventos

francorinque committed:
Merge pull request #73 from Grupo10Henry/franco

estilos a card,estado de redux para los eventos

francorinque committed:
config axios, card events listas por ahora

francorinque committed:
Merge pull request #74 from Grupo10Henry/franco

config axios, card events listas por ahora

lccarrillo247 committed:
Ruta post event arreglada

lccarrillo247 committed:
Ajuste css inputs createvent

lccarrillo247 committed:
Merge pull request #75 from Grupo10Henry/Luis

Ruta post event arreglada

kevinor03 committed:
visualizacion de los banners

kevinor03 committed:
Merge pull request #76 from Grupo10Henry/Kevin

visualizacion de los banners

GuadalupeRedondo committed:
EstilosFAQyContacto

GuadalupeRedondo committed:
Merge pull request #77 from Grupo10Henry/guada

EstilosFAQyContacto

GuadalupeRedondo committed:
CorreccionPalabras

GuadalupeRedondo committed:
Merge pull request #78 from Grupo10Henry/guada

CorreccionPalabras

GuadalupeRedondo committed:
fondoFooter

francorinque committed:
arreglo searchbar, navbar aparece un ruta admin

lccarrillo247 committed:
Tablas Users y Events v01

francorinque committed:
arreglo searchbar, navbar aparece un ruta admin

arreglo searchbar, navbar aparece un ruta admin

kevinor03 committed:
retoque a contact

JE-FUNES committed:
varios

kevinor03 committed:
Merge pull request #80 from Grupo10Henry/Kevin

retoque a contact

lccarrillo247 committed:
Merge pull request #81 from Grupo10Henry/Luis

Tablas Users y Events v01

JE-FUNES committed:
Merge pull request #82 from Grupo10Henry/Juli

varios

JE-FUNES committed:
Merge pull request #83 from Grupo10Henry/develop

Develop

francorinque committed:
navbar aparece en faq

francorinque committed:
navbar aparece en faqnco

navbar aparece en faq

francorinque committed:
arreglo ruta detalle

francorinque committed:
Merge pull request #85 from Grupo10Henry/franco

arreglo ruta detalle

JE-FUNES committed:
Merge pull request #86 from Grupo10Henry/develop

Develop

JE-FUNES committed:
signUp

indicaciones y css

JE-FUNES committed:
Merge pull request #87 from Grupo10Henry/Juli

Juli

francorinque committed:
banner agregado

francorinque committed:
Merge pull request #88 from Grupo10Henry/franco

banner agregado

francorinque committed:
cateogias en carpeta utils

francorinque committed:
Merge pull request #89 from Grupo10Henry/franco

categogias en carpeta utils

lccarrillo247 committed:
Mejoras create event

lccarrillo247 committed:
Merge pull request #90 from Grupo10Henry/Luis

Mejoras create event

lccarrillo247 committed:
Ajustes finales funcionalidad create event

lccarrillo247 committed:
Merge pull request #91 from Grupo10Henry/Luis

Ajustes finales funcionalidad create event

GuadalupeRedondo committed:
Merge branch 'develop' of https://github.com/Grupo10Henry/Client into guada

francorinque committed:
funciones de filtros,dispatch a estado eventsFiltered de redux

francorinque committed:
funciones de filtros,dispatch a estado eventsFiltered de redux

funciones de filtros,dispatch a estado eventsFiltered de redux

GuadalupeRedondo committed:
Merge branch 'develop' of https://github.com/Grupo10Henry/Client into guada

GuadalupeRedondo committed:
UserReviews

GuadalupeRedondo committed:
Merge pull request #93 from Grupo10Henry/guada

Guada

JE-FUNES committed:
SignUp retoques + Detail

JE-FUNES committed:
Merge pull request #94 from Grupo10Henry/Juli

SignUp retoques + Detail

JE-FUNES committed:
Merge pull request #95 from Grupo10Henry/develop

Develop

lccarrillo247 committed:
Mejoras create event

lccarrillo247 committed:
Merge pull request #96 from Grupo10Henry/Luis

Mejoras create event

JE-FUNES committed:
Merge pull request #97 from Grupo10Henry/develop

Develop

lccarrillo247 committed:
Ajustes ceros en minPrice y maxPrice

lccarrillo247 committed:
Merge pull request #98 from Grupo10Henry/Luis

Ajustes ceros en minPrice y maxPrice

JE-FUNES committed:
Merge pull request #99 from Grupo10Henry/develop

Develop

lccarrillo247 committed:
Traer todos los events de la BD al Home

lccarrillo247 committed:
Merge pull request #100 from Grupo10Henry/Luis

Traer todos los events de la BD al Home

JE-FUNES committed:
pj

kevinor03 committed:
modificacion a Recomendados

kevinor03 committed:
Merge branch 'develop' of https://github.com/Grupo10Henry/Client into Kevin

kevinor03 committed:
Merge pull request #101 from Grupo10Henry/Kevin

modificacion a Recomendados

lccarrillo247 committed:
Ajustes para enviar minPrice y maxPrice en 0 en evento no pago

lccarrillo247 committed:
Merge pull request #102 from Grupo10Henry/Luis

Ajustes para enviar minPrice y maxPrice en 0 en evento no pago

JE-FUNES committed:
Merge pull request #103 from Grupo10Henry/develop

Develop

JE-FUNES committed:
js

JE-FUNES committed:
Merge branch 'Juli' of https://github.com/Grupo10Henry/Client into Juli

francorinque committed:
renderizado condicional al filtrar, componente padre de eventos filtrados, estados nuevos en eventsSlice

francorinque committed:
Merge pull request #104 from Grupo10Henry/franco

renderizado condicional al filtrar, componente padre de eventos filtr…

kevinor03 committed:
Merge branch 'develop' of https://github.com/Grupo10Henry/Client into Kevin

GuadalupeRedondo committed:
estilosfooter

GuadalupeRedondo committed:
Merge pull request #105 from Grupo10Henry/guada

estilosfooter

kevinor03 committed:
Merge branch 'develop' of https://github.com/Grupo10Henry/Client into Kevin

francorinque committed:
hook useScrollTo

francorinque committed:
conflico resuelto

francorinque committed:
resuelto conflicto con mostPopular

resuelto conflicto con mostPopular

lccarrillo247 committed:
Creador de secciones v01 (sin crear varios

JE-FUNES committed:
detail en funcionamiento

falta la inserción del mapa y el css, y, cuando esté lo del back, sectores y precios

kevinor03 committed:
funcionalidad de Recomendados

kevinor03 committed:
retoques a componentes de eventos

JE-FUNES committed:
Merge branch 'develop' into Juli

kevinor03 committed:
Merge branch 'develop' into Kevin

kevinor03 committed:
Merge pull request #108 from Grupo10Henry/Kevin

Retoques a los componentes de eventos

kevinor03 committed:
logica de los componentes de eventos

kevinor03 committed:
Merge pull request #109 from Grupo10Henry/Kevin

logica de los componentes de eventos

JE-FUNES committed:
detail, signUp y Login

JE-FUNES committed:
Merge pull request #107 from Grupo10Henry/Juli

Juli

JE-FUNES committed:
Merge pull request #110 from Grupo10Henry/develop

Develop

lccarrillo247 committed:
Merge branch 'develop' of https://github.com/Grupo10Henry/Client into Luis

GuadalupeRedondo committed:
mejorarReviews

GuadalupeRedondo committed:
Merge pull request #111 from Grupo10Henry/guada

mejorarReviews

francorinque committed:
filtros y logica casi terminada

GuadalupeRedondo committed:
reviewsEnHome

francorinque committed:
cambios

JE-FUNES committed:
booking en parte

JE-FUNES committed:
Merge pull request #112 from Grupo10Henry/Juli

Juli

JE-FUNES committed:
Merge pull request #113 from Grupo10Henry/develop

Develop

francorinque committed:
navbar se ve en reserva,carrito

francorinque committed:
Merge pull request #114 from Grupo10Henry/franco

Franco

JE-FUNES committed:
Merge pull request #115 from Grupo10Henry/develop

Develop

JE-FUNES committed:
Cards css

JE-FUNES committed:
Merge pull request #116 from Grupo10Henry/Juli

Juli

kevinor03 committed:
literalmente 5 caracteres

lccarrillo247 committed:
Funcionalidad crear secciones v01

kevinor03 committed:
Merge pull request #117 from Grupo10Henry/Kevin

literalmente 5 caracteres

lccarrillo247 committed:
Merge branch 'develop' of https://github.com/Grupo10Henry/Client into Luis

lccarrillo247 committed:
Mejoras creador de eventos y secciones

lccarrillo247 committed:
Merge pull request #118 from Grupo10Henry/Luis

Luis

GuadalupeRedondo committed:
ReseñasEvento

GuadalupeRedondo committed:
Merge pull request #119 from Grupo10Henry/guada

Guada

JE-FUNES committed:
Merge pull request #120 from Grupo10Henry/develop

Develop

JE-FUNES committed:
countdown

JE-FUNES committed:
Merge pull request #121 from Grupo10Henry/Juli

Juli

francorinque committed:
agrego estado de las fechas

francorinque committed:
estilos a titulos de las secciones

francorinque committed:
estilos a titulos de las secciones

francorinque committed:
Estilos a titulos de las secciones

Estilos a titulos de las secciones

kevinor03 committed:
retoques y preparacion de NextEvents

kevinor03 committed:
Merge pull request #123 from Grupo10Henry/Kevin

retoques y preparacion de NextEvents

lccarrillo247 committed:
Cloudinary para CreateEvent

lccarrillo247 committed:
Merge pull request #124 from Grupo10Henry/Luis

Cloudinary para CreateEvent

francorinque committed:
links para usuarios registrados

francorinque committed:
links user registrado

francorinque committed:
links de user registrado

links de user registrado

francorinque committed:
cambios

francorinque committed:
Merge pull request #126 from Grupo10Henry/franco

cambios

kevinor03 committed:
nextEvent y Contact funcionales

kevinor03 committed:
Merge pull request #127 from Grupo10Henry/Kevin

nextEvent y Contact funcionales

lccarrillo247 committed:
Arreglo conditional chaining en Card y Cards

lccarrillo247 committed:
Merge pull request #128 from Grupo10Henry/Luis

Arreglo conditional chaining en Card y Cards

lccarrillo247 committed:
Botón queda seleccionado en AdminNavbar

lccarrillo247 committed:
Merge pull request #129 from Grupo10Henry/Luis

Botón queda seleccionado en AdminNavbar

JE-FUNES committed:
detail con sectores y precios

JE-FUNES committed:
Merge pull request #130 from Grupo10Henry/Juli

detail con sectores y precios

JE-FUNES committed:
Merge pull request #131 from Grupo10Henry/develop

Develop

JE-FUNES committed:
scrollToTop

con ésto arreglo que las rutas siempre se vean desde arriba

JE-FUNES committed:
Merge pull request #132 from Grupo10Henry/Juli

Juli

JE-FUNES committed:
isDonation condicional en Detail

JE-FUNES committed:
Merge pull request #133 from Grupo10Henry/Juli

isDonation condicional en Detail

JE-FUNES committed:
App con /reserva/:id

JE-FUNES committed:
Merge pull request #134 from Grupo10Henry/Juli

App con /reserva/:id

kevinor03 committed:
alternativa cuando no este el server activo

lccarrillo247 commented:
A presentaaar mañanaaaaa
